src/StateSmith/Output/Gil/C99/C99GenVisitor.cs

```diff
@@ -34,15 +37,23 @@ public C99GenVisitor(SemanticModel model, StringBuilder hFileSb, StringBuilder c
         this.model = model;
         this.hFileSb = hFileSb;
         this.cFileSb = cFileSb;
+        this.renderConfig = renderConfig;
```

Comment:
This isn't used? I think we can remove it?

Reply:
I put all the header/footer code in the VisitCompilationUnit, so this is needed for

`        hFileSb.AppendLineIfNotBlank(renderConfig.FileTop);`

Reply:
oops! I missed that in my initial review. Thanks for explaining. Let's keep it as you have.